cf-agent/cf-agent.c

```diff
@@ -301,9 +304,11 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
      */
     bool cfruncommand = false;
 
-    while ((c = getopt_long(argc_new, argv_new, "tdvnKIf:D:N:VxMB:b:hlC::E", OPTIONS, NULL)) != EOF)
+    while ((c = getopt_long(argc_new, argv_new, "tdvnKIf:D:N:VxMB:b:hlC::ET::",
```

Comment:
Double colon means optional argument, whereas in the long option struct you have put ``required_argument``.